rd_ui/app/scripts/visualizations/cloud.js

```diff
@@ -0,0 +1,505 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g=(g.d3||(g.d3 = {}));g=(g.layout||(g.layout = {}));g.cloud = f()}})(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```

Comment:
Yeah going to need some help with this. I haven't been able to figure out these js package systems.

I modified line 11 below from: 
```javascript
module.exports = function() {
```

Because I couldn't figure out how to reference this module. I'm not sure where that module structure even comes from D3, Angular? Been digging into both but not finding any docs that have been illuminating yet.

I did find the bower docs and add `d3-cloud` this to the `rd_ui/bower.json` but can't figure out how to reference it. 

I tried `var cloud = require("d3-cloud")` but that is throwing up errors: `angular.js:9937 ReferenceError: require is not defined`

Directly referencing `module.exports` in the same way seems like a mistake.